src/fastertransformer/core/Buffer.h

```diff
@@ -66,7 +66,7 @@ class Buffer {
     inline T* data() const {
         static_assert(!std::is_same<T, void>::value);
         FT_CHECK_WITH_INFO(
-            type_ == getTensorType<T>(),
+            type_ == getTensorType<T>() || type_ == DataType::TYPE_INT4X2,
```

Comment:
Don't change this, to get data pointer for quantized buffer, please use non-template version of data() and cast manually.

Reply:
Fixed. Thanks~